docs/content/docs/tutorials/emr-on-eks-example.md

```diff
@@ -0,0 +1,494 @@
+aws iam create-policy   \
+  --policy-name ack-${SERVICE}-jobexecution-policy \
+  --policy-document file://job_policy.json
+
+echo -n "Attaching IAM policy ..."
+aws iam attach-role-policy \
+  --role-name "${ACK_JOB_EXECUTION_ROLE}" \
+  --policy-arn "arn:aws:iam::${AWS_ACCOUNT_ID}:policy/ack-${SERVICE}-jobexecution-policy"
+
+aws emr-containers update-role-trust-policy \
+  --cluster-name ${EKS_CLUSTER_NAME} \
+  --namespace ${EMR_NAMESPACE} \
+  --role-name ${ACK_JOB_EXECUTION_ROLE}
+```
+Now let's submit sample spark job
+```
+echo "checking if VirtualCluster Status is "True""
+VC=$(kubectl get virtualcluster -o jsonpath='{.items..metadata.name}')
+kubectl describe virtualcluster/$VC | yq e '.Status.Conditions.Status'
+
+echo "get VirtualCluster ID"
+export VC_ID=$(kubectl describe virtualcluster/$VC | yq e '.Status.Id')
+
+export RANDOM_ID=$(LC_ALL=C tr -dc a-z0-9 </dev/urandom | head -c 8)
+
+cat << EOF > jobrun.yaml
+---
+apiVersion: emrcontainers.services.k8s.aws/v1alpha1
+kind: JobRun
+metadata:
+  name: my-ack-jobrun-${RANDOM_ID}
+spec:
+  name: my-ack-jobrun-${RANDOM_ID}
+  virtualClusterID: "${VC_ID}"
+  executionRoleARN: "${ACK_JOB_EXECUTION_ROLE_ARN}"
+  releaseLabel: "emr-6.3.0-latest"
+  jobDriver:
+    sparkSubmitJobDriver:
+      entryPoint: "local:///usr/lib/spark/examples/src/main/python/pi.py"
+      entryPointArguments:
+      sparkSubmitParameters: "--conf spark.executor.instances=2 --conf spark.executor.memory=1G --conf spark.executor.cores=1 --conf spark.driver.cores=1"
+EOF
+```
+```
+echo "running sample job"
+kubectl apply -f jobrun.yaml
+kubectl describe jobruns
+```
+**Expected outcome**
+```
+Name:         my-ack-jobrun-t2rpcpks
+Namespace:    default
+Labels:       <none>
+Annotations:  <none>
+API Version:  emrcontainers.services.k8s.aws/v1alpha1
+Kind:         JobRun
+...
+Status:
+  Ack Resource Metadata:
+    Arn:               arn:aws:emr-containers:us-west-2:012345678910:/virtualclusters/dxnqujbxexzri28ph1wspbxo0/jobruns/000000030mrd934cdqc
+    Owner Account ID:  012345678910
+    Region:            us-west-2
+  Conditions:
+    Last Transition Time:  2022-08-26T18:29:12Z
+    Message:               Resource synced successfully
+    Reason:                
+    Status:                True
+    Type:                  ACK.ResourceSynced
+  Id:                      000000030mrd934cdqc
+Events:                    <none>
+```
+
+## Cleanup
+Simply run these commands to cleanup your environment
+```
+# delete all custom resources
+kubectl delete -f virtualcluster.yaml
+# note: you cannot delete jobruns unless you want to cancel them
+
+# uninstall emrcontainers controller
+helm delete ack-$SERVICE-controller -n $ACK_SYSTEM_NAMESPACE
+
+# delete crds
+kubectl delete crd/virtualclusters.emrcontainers.services.k8s.aws
+kubectl delete crd/jobruns.emrcontainers.services.k8s.aws
+
+# delete namespace
+kubectl delete ns $ACK_K8S_NAMESPACE
+kubectl delete ns $EMR_NAMESPACE
```

Comment:
You can also just delete the namespace or use something like `kubectl delete all --all -n {namespace}`

Reply:
added in next commit